draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -366,7 +373,9 @@ public boolean isClosed() {
     if (activePointerId == INVALID_POINTER) {
       return false;
     }
-    viewDragHelper.processTouchEvent(ev);
+    if(getContext().getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT){
+      viewDragHelper.processTouchEvent(ev);
+    }
```

Comment:
Definitively this should be configurable. You should add a ``onlyPortrait`` flag in to the ``attrs.xml`` file and use this value here.